skada/_subspace.py

```diff
@@ -271,7 +273,7 @@ def fit(self, X, y=None, sample_domain=None, **kwargs):
         B = K @ H @ K
         solution = np.linalg.solve(A, B)
 
-        eigvals, eigvects = np.linalg.eigh(solution)
+        eigvals, eigvects = np.linalg.eig(solution)
```

Comment:
this means that evals can be complex. Is that a problem?

Reply:
ho, this is a mistake, I shouldn't be changing this